packages/eddsa-proof/src/generate.ts

```diff
@@ -30,7 +29,7 @@ export default async function generate(
     // If the Snark artifacts are not defined they will be automatically downloaded.
     /* istanbul ignore next */
     if (!snarkArtifacts) {
-        snarkArtifacts = await getSnarkArtifacts()
+        snarkArtifacts = await getEddsaSnarkArtifacts()
```

Comment:
Maybe better `EdDSA`? just to make it consistent with the same occurrence in the other packages.